openedx/features/course_card/views.py

```diff
@@ -1,25 +1,33 @@
+"""
+Views file for Course Card Application
```

Comment:
All views for course card application

By the way python call it module instead of file

Reply:
right thanks